spec/integration/logstash_forwarder_spec.rb

```diff
@@ -1,108 +0,0 @@
-# encoding: utf-8
```

Comment:
shouldn't be build a version of tests for lumberjack?

Reply:
I would support adding plugin level changes here and cover Lumberjack integration tests in the core like [we have beats](https://github.com/mashhurs/logstash/blob/19637143e6ef24b7b4009e71b1fa32b04b92c942/qa/integration/specs/beats_input_spec.rb#L26). That would cover `lumberjack -> beats` and `lumberjack -> lumberjack` use cases. I don't see any lumberjack integration tests but we [suggest it to LS-to-LS comm](https://www.elastic.co/guide/en/logstash/current/ls-to-ls-lumberjack.html) 🤦